core/src/main/java/io/confluent/connect/storage/StorageSinkConnectorConfig.java

```diff
@@ -61,9 +61,9 @@
   public static final String APPEND_LATE_DATA = "append.late.data";
   public static final Boolean APPEND_LATE_DATA_DEFAULT = false;
   public static final String APPEND_LATE_DATA_DOC = "If true, the partitions "
-      + "will append data from previous (already commited) partitions into the current, open"
-      + " partition. If false, late data intended for closed partitions will be rolled into "
-      + "files marked with the partition indicated by the extracted time, even if this creates"
+      + "will append data from previous (already commited) partitions into the current, open "
```

Comment:
typo remains: `commited` -> `committed`. But based on discussion in the other PR, this description needs to be improved.